test/higher_order_ops/test_invoke_subgraph.py

```diff
@@ -475,7 +475,30 @@ def fn(x, y):
 
         opt_fn = torch.compile(fn, backend="inductor", fullgraph=True)
         with self.assertRaisesRegex(
-            torch._dynamo.exc.Unsupported, "NYI: invoke_subgraph with aliasing"
+            torch._dynamo.exc.Unsupported,
+            "Encountered input mutation during higher order op tracing",
```

Comment:
can this have the name of the HOP in it? (or does it already?)

Reply:
Added.